test/appium/views/base_view.py

```diff
@@ -555,12 +555,15 @@ def relogin(self, password=common_password):
         sign_in_view = self.get_sign_in_view()
         sign_in_view.sign_in(password)
 
+    def close_share_chat_key_popup(self):
+        TouchAction(self.driver).tap(None, 255, 104, 1).perform()
+
```

Comment:
Probably there is a better way to implement this (it should tap outside popup)

Reply:
We could use just swipe_down for this purpose but this one is fine too.